client/rpc/staking_client.go

```diff
@@ -16,7 +16,7 @@ import (
 	coretypes "github.com/tendermint/tendermint/rpc/core/types"
 )
 
-const BscSideChainId = "bsc"
+const SideChainId = "rialto"
```

Comment:
`rialto` is the network name of our first testnet, there may be more testnet, and will have main-net.  So I think you can  use `const RialtoChainId = "rialto"` here, and `RialtoChainId` may use in many place rather than staking module, you may place it somewhere more common.
`